toonz/sources/toonz/startuppopup.cpp

```diff
@@ -0,0 +1,889 @@
+                         ->getCurrentScene()
+                         ->getScene()
+                         ->getProject()
+                         ->getScenesPath()
+                         .getQString());
+  TDimensionD cameraSize = TApp::instance()
+                               ->getCurrentScene()
+                               ->getScene()
+                               ->getCurrentCamera()
+                               ->getSize();
+  TDimension cameraRes = TApp::instance()
+                             ->getCurrentScene()
+                             ->getScene()
+                             ->getCurrentCamera()
+                             ->getRes();
+  double fps = TApp::instance()
+                   ->getCurrentScene()
+                   ->getScene()
+                   ->getProperties()
+                   ->getOutputProperties()
+                   ->getFrameRate();
+  m_widthFld->setValue(cameraSize.lx);
+  m_heightFld->setValue(cameraSize.ly);
+  if (Preferences::instance()->getCameraUnits() == "pixel") {
+    m_widthFld->setDecimals(0);
+    m_heightFld->setDecimals(0);
+    m_resTextLabel->hide();
+    m_resXFld->hide();
+    m_resYFld->hide();
+    m_resXLabel->hide();
+    m_dpiFld->hide();
+    m_dpiLabel->hide();
+  } else {
+    m_widthFld->setDecimals(4);
+    m_heightFld->setDecimals(4);
+    m_resXFld->show();
+    m_resYFld->show();
+    m_resXLabel->show();
+    m_resTextLabel->show();
+    m_dpiFld->show();
+    m_dpiLabel->show();
+  }
+
+  m_fpsFld->setValue(fps);
+  m_unitsCB->setCurrentText(Preferences::instance()->getCameraUnits());
+  m_dpi  = cameraRes.lx / cameraSize.lx;
+  m_xRes = cameraRes.lx;
+  m_yRes = cameraRes.ly;
+  m_resXFld->setValue(m_xRes);
+  m_resYFld->setValue(m_yRes);
+  m_resXFld->setDecimals(0);
+  m_resYFld->setDecimals(0);
+  m_dpiFld->setValue(m_dpi);
+
+  int boxWidth  = m_sceneBox->width();
+  int boxHeight = m_sceneBox->height();
+  m_sceneBox->setFixedWidth(boxWidth);
+  m_projectBox->setFixedWidth(boxWidth);
+  m_recentBox->setMinimumHeight(boxHeight);
+
+  // update recent scenes
+  // clear items if they exist first
+
+  if (m_recentSceneLay->count() > 0) {
+    QLayoutItem *child;
+    while (m_recentSceneLay->count() != 0) {
+      child = m_recentSceneLay->takeAt(0);
+      delete child;
+    }
+  }
+
+  m_sceneNames = RecentFiles::instance()->getFilesNameList(RecentFiles::Scene);
+  m_recentNamesLabels = QVector<StartupLabel *>(m_sceneNames.count());
+
+  if (m_sceneNames.count() <= 0) {
+    m_recentSceneLay->addWidget(new QLabel(tr("No Recent Scenes"), this), 1,
+                                Qt::AlignTop);
+  } else {
+    int i = 0;
+    for (QString name : m_sceneNames) {
+      if (i > 9) break;  // box can hold 10 scenes
+      QString justName = QString::fromStdString(TFilePath(name).getName());
+      m_recentNamesLabels[i] = new StartupLabel(justName, this, i);
+      m_recentSceneLay->addWidget(m_recentNamesLabels[i], i, Qt::AlignTop);
```

Comment:
How about to changing background color of `StartupLabel` on mouse hover, in order to make it clear for users that these "Open Recent Scene" items are clickable ?

<img src="https://cloud.githubusercontent.com/assets/17974955/19433885/99a4f58e-949d-11e6-9136-1d9ce4698260.png" width=600>

(As it is not a bug, I can post this mock-up as a separate PR after this PR is merged.)